csharp_dotnetcore/8.Suggested-Actions/Suggested_Actions/SuggestedActionsBot.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+
+namespace Suggested_Actions
+{
+    /// <summary>
+    /// This bot will respond to the user's input suggested actions.
```

Comment:
This comment means something *ONLY* if I already know what input suggested actions are. In this case, I think we should add more here.